lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -254,25 +251,27 @@ data FollowExit
 --
 -- Exits with 'Nothing' in case of error.
 follow
-    :: forall block e. (Show e)
+    :: forall block e tr. (Show e)
```

Comment:
Perhaps `tr` -> `msg` ?

Reply:
Kind of liked the `tr`, but I guess that makes sense as `Tracer IO tr` would appear like a tracer of tracer 👍 

Adding a commit in the other PR.